### PR TITLE
Mods for file input and output

### PR DESCRIPTION
**Pull request by 0ces**
# Added file I/O support
- <-f inputfile.txt>  _reads input from a txt file and fetches each email_
- <-o output.json> _outputs all the data collected to a json file_

### CODE REVIEW

**Review by iheind — CHANGES_REQUESTED** [review]
hi thank you. there error in append location, but i fix with this.
`                loc_names = []
                for loc in locations:
                    loc_names.append(
                        f"- {loc['avg']['town']}, {loc['avg']['country']}"
                    )

                loc_names = set(loc_names)  # delete duplicates
                #output['locations'] = loc_names
                for loc in loc_names:
                    print(loc)
                
                output['locations'] = loc

File: hunt.py [hunk 1]
```diff
@@ -22,163 +22,185 @@
...
 
 if __name__ == "__main__":
-
     banner()
-    
     # We change the current working directory to allow using GHunt from anywhere
     os.chdir(Path(__file__).parents[0])
-
     if len(sys.argv) <= 1:
         exit("Please put an email address.")
```
Comment (pattylee016) [inline hunt.py:32]: zaidepaullucas@gmail.com

**Review by pattylee016 — CHANGES_REQUESTED** [review]

File: hunt.py (hunk 1 above)
Comment (kghd00) [reply hunt.py:32]: Hgsanders81@gmail.com

Comment (mxrch) [reply hunt.py:32]: Why are you commenting email adresses ?

Comment (0ces) [reply hunt.py:32]: No idea

Comment (kghd00) [reply hunt.py:32]: all honestest , I have no idea what I'm doing on here. My apologies. I would love to use this email thing you got going on tho.

Comment (0ces) [reply hunt.py:32]: Just clone the repository and follow the instructions for the requirements and usage, good luck

**Review by xi2lo — APPROVED** [review]

**Review by Flyboy95 — APPROVED** [review]

**Review by smt8888 — APPROVED** [review]

**Review by mrmarley5 — APPROVED** [review]

**Review by mrmarley5 — APPROVED** [review]

**Comment by mxrch** [comment]
Sorry, I let this PR too much long and it's no more compatible with all the changes today, but I'll think about introducing JSON output.
Thanks for your contribution.

**Pull request closed without merging**